allennlp/data/token_indexers/pretrained_transformer_indexer.py

```diff
@@ -35,6 +37,7 @@ def __init__(self, model_name: str, namespace: str = "tags", **kwargs) -> None:
         super().__init__(**kwargs)
         self._namespace = namespace
         self._tokenizer = AutoTokenizer.from_pretrained(model_name)
+        self._padding_value = self._tokenizer.convert_tokens_to_ids([self._tokenizer.pad_token])[0]
```

Comment:
Paranoia: Could you assert that you only get back a single item?

Reply:
Actually, that's no longer needed. You can just call `tokenizer.pad_token_id`. I changed it that way.

Reply:
e132b61ec3672f56fb06eb1cb8a80ae3cdff7426

Reply:
👍